### ISSUE TITLE
Rating component missing aria-label

### ISSUE THREAD

**Issue by shainanigans** [valor-software/ng2-bootstrap#5579]
**Bug description:**
The rating component is missing an `aria-label` attribute on the `span` with `role="slider"`. This was discovered using the axe Chrome extension. More information about the issue can be found [here](https://dequeuniversity.com/rules/axe/3.4/aria-input-field-name?application=AxeChrome).

**Example:**
Inspecting the [demo](https://valor-software.com/ngx-bootstrap/#/rating#rating-basic) in the docs, you can see the issue with the aforementioned `span` element.

**Expected behavior**
The element should have an `aria-label` attribute on it. Ideally, this would be an input so that developers can add the most informative label for their use case.

**Comment by shainanigans** [comment]
Hi, could someone please address this issue?

**Comment by IraErshova** [comment]
@shainanigans Thanks for your issue. I'm going to start working on it right now!

**Comment by shainanigans** [comment]
Thanks @IraErshova !
